development/lint/eslint-rule-enforce-return-type.js

```
@@ -0,0 +1,44 @@
+const enforceReturnTypeRule = {
+  meta: {
+    type: 'problem',  
```

ESLint

[error] Delete `··`

(prettier/prettier)